File: ARCHIVE/uri/1-Beginner/2310.c
#include <stdio.h>

int i;
double runs, ser_amt, ser_suc, blc_amt, blc_suc, att_amt, att_suc;
double total[3], sucess[3];
char name[1000];

int main (void){

	scanf("%lf", &runs);

	for (i = 0; i < runs; i++){

		scanf("%s", name);

		scanf("%lf %lf %lf", &ser_amt, &blc_amt, &att_amt);
		scanf("%lf %lf %lf", &ser_suc, &blc_suc, &att_suc);

		total[0] += ser_amt;
		total[1] += blc_amt;
		total[2] += att_amt;

		sucess[0] += ser_suc;
		sucess[1] += blc_suc;
		sucess[2] += att_suc;

	}

	printf("Pontos de Saque: %.2lf %%.\n", (sucess[0] / total[0]) * 100);
	printf("Pontos de Bloqueio: %.2lf %%.\n", (sucess[1] / total[1]) * 100);
	printf("Pontos de Ataque: %.2lf %%.\n", (sucess[2] / total[2]) * 100);

	return 0;
}

File: ARCHIVE/2018-03-28-2100/2310.py
n = input()
a = 0
b = 0
c = 0
a1 = 0
b1 = 0
c1 = 0
for i in range (0, n):
    name = raw_input ()
    f = raw_input ()
    f = f.split (' ')
    s = raw_input ()
    s = s.split (' ')

    a += float (f[0])
    b += float (f[1])
    c += float (f[2])

    a1 += float (s[0])
    b1 += float (s[1])
    c1 += float (s[2])

print "Pontos de Saque: %0.2lf %%." % (a1*100/a)
print "Pontos de Bloqueio: %0.2lf %%." % (b1*100/b)
print "Pontos de Ataque: %0.2lf %%." % (c1*100/c)
